llvm/lib/Target/AMDGPU/AMDGPUInstructionSelector.h

```diff
@@ -353,7 +353,6 @@ class AMDGPUInstructionSelector final : public InstructionSelector {
   void renderFPPow2ToExponent(MachineInstrBuilder &MIB, const MachineInstr &MI,
                               int OpIdx) const;
 
-  bool isInlineImmediate16(int64_t Imm) const;
```

Comment:
I see this function as well as isInlineImmediate32 and isInlineImmediate64 are called in
AMDGPUGenGlobalISel.inc AMDGPUInstructionSelector::testImmPredicate_I64, but I'm not sure if that function is used. 

Perhaps you can remove all 3 functions?